### ISSUE TITLE
save-all path navigation issue

### ISSUE THREAD

**Issue by emanruse** [newsboat/newsboat#1418]
### STR:

0. Have a directory structure:
```
/home/username/saved-articles/somedir/
```

1. Set in config:

```
save-path "~/saved-articles/"
```

2. Start newsboat, open a feed and activate the save-all function
3. Newsboat shows:

```
Directory: /home/username/saved-articles
```
4. Navigate to `/home/username/saved-articles/somedir` using arrow keys and press Enter

### Expected:

Newsboat should display and attempt to use `/home/username/saved-articles/somedir` as requested.

### Actual:

Newsboat shows and attempts to use `/home/username/saved-articles/somedir/saved-articles`, i.e. it adds a trailing "saved-articles" to the target directory string and one must go and manually delete it before confirming, otherwise the saving fails.

**Comment by emanruse** [comment]
```
newsboat 2.21.0 - https://newsboat.org/
Copyright (C) 2006-2015 Andreas Krennmair
Copyright (C) 2015-2020 Alexander Batischev
Copyright (C) 2006-2017 Newsbeuter contributors
Copyright (C) 2017-2020 Newsboat contributors

Newsboat is free software licensed under the MIT License. (Type `newsboat -vv' to see the full text.)
It bundles:
- JSON for Modern C++ library, licensed under the MIT License: https://github.com/nlohmann/json
- optional-lite library, licensed under the Boost Software License: https://github.com/martinmoene/optional-lite
- expected-lite library, licensed under the Boost Software License: https://github.com/martinmoene/expected-lite

newsboat 2.21.0
System: Linux 5.10.1-1-pae (i686)
Compiler: g++ 10.2.1 20200825 [revision c0746a1beb1ba073c7981eb09f55b3d993b32e5c]
ncurses: ncurses 6.2.20201205 (compiled with 6.2)
libcurl: libcurl/7.74.0 OpenSSL/1.1.1h-fips zlib/1.2.11 brotli/1.0.9 zstd/1.4.5 libidn2/2.3.0 libpsl/0.21.1 (+libidn2/2.3.0) libssh/0.9.5/openssl/zlib nghttp2/1.41.0 (compiled with 7.73.0)
SQLite: 3.34.0 (compiled with 3.33.0)
libxml2: compiled with 2.9.10
```

**Comment by Minoru** [comment]
Thank you for the report! @dennisschagt fixed it in #1421, so Newsboat 2.23 won't have this bug anymore.
